pkg/policy/policy.go

```diff
@@ -42,6 +44,21 @@ type Check struct {
 	Reason       string    `hcl:"reason,optional"`
 }
 
+type Analytic struct {
+	// Whether policy will persist in database
+	Persistence bool
+	// Whether policy was defined in configuration
+	UsedConfig bool
```

Comment:
[non blocking] What are the other options to define policies? Via a CLI argument?

How about changing it to `UsageOrigin` or something like that? Then it can have multiple values.

Not this this is very relevant to feel free to ignore

Reply:
Yeah I can change the name no problem, good idea